python/ray/_raylet.pyx

```diff
@@ -935,6 +935,34 @@ cdef class CoreWorker:
 
         return RayObjectsToDataMetadataPairs(results)
 
+    def kv_put(self, c_string key, c_string value):
+        with nogil:
+            check_status(CCoreWorkerProcess.GetCoreWorker().KVPut(key, value))
+
+    def kv_del(self, c_string key):
+        with nogil:
+            check_status(CCoreWorkerProcess.GetCoreWorker().KVDel(key))
+
+    def kv_get(self, c_string key):
```

Comment:
This means we can use these only within ray workers (driver + worker)? As far as I remember, there are components that are not parts of Ray core components but use these values.